process/track/blockProcessor.go

```diff
@@ -231,7 +231,7 @@ func (bp *blockProcessor) ComputeLongestChain(shardID uint32, header data.Header
 	var sortedHeadersHashes [][]byte
 
 	defer func() {
-		bp.requestHeadersIfNeeded(header, sortedHeaders, headers)
+		go bp.requestHeadersIfNeeded(header, sortedHeaders, headers)
```

Comment:
I think there is a go routine in that function - thus there is no need for this. or delete go routine in the underlying funciton

Reply:
Actually I have added because the call of the new method AddHeaderFromPool which has been added in underlying method. I will remove go from there